addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -81,7 +81,8 @@ def get_update_included_pdf_params(self):
                     'files': [{
                         'name': doc.name.rstrip('.pdf'),
                         'id': doc.id,
-                        'is_selected': doc in line.product_document_ids,
+                        'is_selected': doc in line.sudo().product_document_ids, # User should be
+                        # able to access all product documents even without sales access
```

Comment:
doc is then in sudo, and being returned inside the dialog_params, we'll avoid that. I don't think that sudo is needed though, that field doesn't have any specific access.